Tasks/Xcode/task.json

```diff
@@ -12,7 +12,7 @@
     "version": {
         "Major": 2,
         "Minor": 0,
-        "Patch": 2
+        "Patch": 3
```

Comment:
Let's change this to 4 instead. Trying out a patch for onedrive with patch version 3.